addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -657,47 +657,41 @@ QUnit.test("Unfollow message", async function () {
     await contains(".o-mail-Message", { count: 3 });
     await click(":nth-child(1 of .o-mail-Message) button[title='Expand']");
     await contains(":nth-child(1 of .o-mail-Message)", {
-        contains: [
-            [".o-mail-Message-header small", { text: "on Thread followed" }],
-            [".o-mail-Message-moreMenu"],
-            ["span[title='Unfollow']"],
-        ],
+        contains: [[".o-mail-Message-header small", { text: "on Thread followed" }]],
     });
+    await contains(".o-mail-Message-moreMenu", { count: 1 });
```

Comment:
Hello, the idea of the PR looks good, just a quick comment as I'm passing by. 
Why splitting the contains here? The goal was to check all the parts are present on the same parent at the same time, now this guarantee is removed here. Test probably won't pass as there are multiple more menu in DOM anyway.

Reply:
The dropdown's content is now at the bottom of the DOM, I've added further explanations in the PR's main comment. Some elements were thus not contained in the selected parent anymore but I agree that some selectors should be more specific, I'll try to improve them.